src/apps/config/channel.lua

```diff
@@ -27,7 +27,7 @@ local ring_buffer_t = ffi.typeof([[struct {
 -- A: We need a variable-sized mapping.
 local function create_ring_buffer (name, size)
    local path = shm.resolve(name)
-   shm.mkdir(path)
+   shm.mkdir(lib.dirname(path))
```

Comment:
This fix should apply to the merge in a separate patch, just to get the test suite working after the merge with no changes of ours on top